packages/dds/tree/src/test/utils.ts

```diff
@@ -789,14 +791,23 @@ const sf = new SchemaFactory("com.fluidframework.json");
 export const NumberArray = sf.array("array", sf.number);
 export const StringArray = sf.array("array", sf.string);
 
+export const IdentifierSchema = sf.object("identifier-object", {
+	identifier: sf.identifier,
+});
+
 const jsonPrimitiveSchema = [sf.null, sf.boolean, sf.number, sf.string] as const;
 export const JsonObject = sf.mapRecursive("object", [
 	() => JsonObject,
 	() => JsonArray,
 	...jsonPrimitiveSchema,
 ]);
+export const jsonObjectSchemaIdentifier: TreeNodeSchemaIdentifier = brand(
```

Comment:
nit: It wouldn't be the worst thing to just get rid of `jsonObjectSchemaIdentifier` and have users of it do `brand(JsonObject.identifier)`. But this is fine too, although maybe it makes a tiny bit more sense to do:

```ts
export const jsonObjectSchemaIdentifier: TreeNodeSchemaIdentifier = brand("object");
export const JsonObject = sf.mapRecursive(jsonObjectSchemaIdentifier, ...);
```